lua/nvim-treesitter/install.lua

```diff
@@ -192,6 +206,13 @@ M.commands = {
       "-nargs=+",
       "-complete=custom,v:lua.ts_installable_parsers"
     }
+  },
+  TSUpdate = {
+    run = M.update,
+    args = {
+      "-nargs=*",
+      "-complete=custom,v:lua.ts_installable_parsers"
```

Comment:
Can't we create a custom completion based on `installed_parsers` ?

Reply:
I added a completion based on installed parsers.